Providers/Profile.swift

```diff
@@ -374,6 +389,15 @@ open class BrowserProfile: Profile {
                                       date: Date().toMicrosecondsSince1970(),
                                       type: visitType)
                 history.addLocalVisit(visit)
+                if self.historyApiConfiguration == .new {
+                    let result = self.places.applyObservation(visitObservation: VisitObservation(url: url.description, title: title as String, visitType: VisitTransition.fromVisitType(visitType: visitType)))
```

Comment:
nit, we try to cut line length down so we can enable linter rules for it in the future. We try to monitor new code for that. I won't make the comments for other long lines so this doesn't get repetitive.
```suggestion
                    let result = self.places.applyObservation(
                        visitObservation: VisitObservation(url: url.description,
                                                           title: title as String,
                                                           visitType: VisitTransition.fromVisitType(visitType: visitType))
                    )
```